packages/ui-view/src/View/styles.ts

```diff
@@ -600,16 +609,16 @@ const generateStyle = (
       ...(shouldUseFocusStyles ? focusStyles : {})
     },
     inlineStyles: {
-      ...spacingStyle,
-      ...borderStyle,
-      ...offsetStyle,
-      width,
-      height,
-      minWidth,
-      minHeight,
-      maxWidth,
-      maxHeight,
-      ...getStyleProps(props)
+      ...important(spacingStyle),
+      ...important(borderStyle),
+      ...important(offsetStyle),
+      width: `${width} !important`,
+      height: `${height} !important`,
+      minWidth: `${minWidth} !important`,
+      minHeight: `${minHeight} !important`,
+      maxWidth: `${maxWidth} !important`,
+      maxHeight: `${maxHeight} !important`,
+      ...important(getStyleProps(props))
```

Comment:
I would wrap the whole inlineStyles into the important method instead of one-by-one. It is needed too, because width and height etc. can be undefined too, and it would result in invalid `undefined !important`. I tried it out, and almost halves the render time too.